### ISSUE TITLE
[Bright Wizard] Flashfire

### ISSUE THREAD

**Issue by Morwar** [WarEmu/WarBugs#18400]
I've been playing my BW for a while and every time I run Flashfire I can see that sometimes my Fireballs build up for 50% of the build up time and sometimes I cast them on instant. After a while I realised that it is due to whether you move or you stand still. When your Flashfire procs and you cast a Fireball while standing it will trigger a 50% build up time reduction, whereas you cast it while moving it will trigger instant Fireball. Here are some two screenshots to show that:

Standing still (with cast bar and taking damage once the projectile reaches the target):
![Morney_010](https://user-images.githubusercontent.com/10424820/105647456-bc7b3680-5e9d-11eb-856d-b1833f8485e4.jpg)

While moving (no cast bar and instant effect):
![Morney_011](https://user-images.githubusercontent.com/10424820/105647475-dd438c00-5e9d-11eb-8137-9a7a3d9987d9.jpg)

Thanks,
Mor

**Comment by CornerbackAC** [comment]
Error still happens

**Comment by Morwar** [comment]
> Error still happens

Yep, you still need to move while casting spells with Flashfire to make them instant.
